### DISCUSSION TITLE
SLZB-06M as a router

### DISCUSSION THREAD

**Discussion by DJBenson** [Koenkk/zigbee2mqtt#24828] [Devices]
In #21339 this device was added as a router. The instructions say to click the "router reconnect" button in the SLZB interface but doesn't say what other steps are required. Does the device need including into my Z2M network? I've enabled join mode on my coordinator but the device never pairs but I've no idea if that's expected or not...

Running router firmware 20231130 on the device and Z2M version 1.41.0.

**Comment by chanchanbzh** [comment]
Hello everyone, I have the same problem as you.

I have a Z2M instance (1.41.0) with a Conbee II as coordinator in my living room (in red on the image).
This instance is used by the ZigbeeLinker plugin of Jeedom which controls all the home automation of my house and which publishes on the topic "zigbee2mqtt".

I am trying to include a SLZB-06M (Core => v2.6.8.dev16 and Radio => 20231130) as a router which is in the boiler room on the other side of my house (in blue on the image), to extend the network so that it can go to my shed (I live in a small farm so I have long distances).
Unable to include the SLZB-06M when it is in its final place in the boiler room. FYI, right next to it, I have connected sockets that are perfectly included and controllable in Z2M.
I had to put my SLZB-06M right next to my Conbee II key in my living room to be able to include it in Z2M.
When I put it back in the boiler room, it is no longer reachable.

I have a 2nd Z2M instance with a 2nd SLZB-06M as coordinator in my hangar (in green on the image).
This 2nd instance is connected to Jeedom thanks to the jMQTT plugin and I recover my devices by subscribing to the topic "zigbee2mqtt_hangar".

Yesterday, I added a Switch module (ZBMINIR2) on the 2nd Z2M instance and surprisingly, I observed that my SLZB-06M from my Boiler Room was included in this instance!

By default, this operation suits me but I would have preferred to have only one Z2M instance (the one in red) and that the SLZB-06M of my boiler room (in blue) acts as a router with the SLZB-06M of my hangar (in green) in order to centralize all my Zigbee devices in the same place in the ZigbeeLinker plugin.

If someone can help us.
Thanks in advance.
Sorry for the translation, I am in France and I do not speak English (thanks google translate)
Chanchan BZH
![@_Sans titre](https://github.com/user-attachments/assets/bb747212-776f-411b-ab0b-e655869eaf35)
